### PR TITLE
Publisher stats API docs

### PR DESCRIPTION
**Pull request by fraczles**
Edit re: https://github.com/hellonarrativ/api-docs/pull/48#issuecomment-394376383

Made the docs more consistent with other API pages

image

image

### CODE REVIEW

File: docs/statistics.rst [hunk 1]
```diff
@@ -0,0 +1,144 @@
+Statistics API
+----------------------------
+
+Fetches statistics for a given publisher over a specified date range.
+
+-  **URL**
```
Comment (myfashionhub) [inline docs/statistics.rst]: I don't think these need to be bullet points?

**Comment by csdev** [comment]
Style is inconsistent with the other API endpoint documentation pages. If you think that the format should be changed, then let's discuss that separately, but that should not be in the scope of this PR.

File: docs/statistics.rst [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+     - **Required**: The starting date to collect statistics, as a ISO 8601 compliant string. Ex:
+       '2018-01-01', '2015-03-11', '2016'
+
+   * - date_to
+     - string
+     - **Required**: The end date to collect statistics, as a ISO 8601 compliant string. Must be greater than `date_from`.
+
+   * - limit
+     - integer
+     - Optional: The upper limit on the number of rows returned by this query.
```
Comment (PhilWinchester) [inline docs/statistics.rst]: There's a default value of 5k rows on the query. Is it worth specifying that we have a default value in the docs?

Comment (fraczles) [reply docs/statistics.rst]: Yeah, that seems like a good idea to me. Nice catch 👍

File: docs/statistics.rst [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+   * - advertiser_name
+     - string
+     - The full name of the advertiser.
+
+   * - edit_name
+     - string
+     - The full name of the edit.
+
+   * - advertiser_attributed_sales
+     - float
```
Comment (csdev) [inline docs/statistics.rst]: int

File: docs/statistics.rst [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+     - The total number of sales attributed to the advertiser.
+
+   * - pub_earnings
+     - float
```
Comment (csdev) [inline docs/statistics.rst]: string (also put a note here on format?)

File: docs/statistics.rst [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+   * - merch_id
+     - integer
+     - The id of the merchant.
+
+   * - impressions
+     - integer
+     - The total number of page impressions.
+
+   * - advertiser_attributed_revenue
+     - float
```
Comment (csdev) [inline docs/statistics.rst]: string

File: docs/statistics.rst [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+     - The total number of sales attributed to the advertiser.
+
+   * - pub_earnings
+     - float
+     - The amount of money earned by the publisher in USD.
+
+   * - edit_id
+     - integer
+     - The id of the particular edit.
```
Comment (csdev) [inline docs/statistics.rst]: "identifier" or "ID", not "id"

File: docs/statistics.rst [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+
+Examples
+^^^^^^^^
+
+
+Get statistics grouped by edits and merchants per day between 2018-03-11 and 2018-03-12::
+
+
+    GET /api/v1/publishers/1/stats_by_edit_merchant_daily/
```
[the remark this answers is not in the document]
Comment (fraczles) [reply docs/statistics.rst]: I see other URL pages just including the path:

- image

- image

**Pull request merged**